client-common/src/main/java/org/apache/cassandra/sidecar/common/response/GossipStatusResponse.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.cassandra.sidecar.common.response;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+/**
+ * Response for GET gossip status API
+ */
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class GossipStatusResponse
+{
+    /**
+     * Constructs a response with gossip status
+     * @param gossipRunning running status of gossip
+     */
+    public GossipStatusResponse(@JsonProperty("gossipRunning") boolean gossipRunning)
+    {
+        this.gossipRunning = gossipRunning;
+    }
+
+    @JsonProperty("gossipRunning")
+    public boolean gossipRunning()
+    {
+        return gossipRunning;
+    }
+
+    private final boolean gossipRunning;
```

Comment:
Define this at the beginning of the class

Reply:
Most of Sidecar request/response classes following this style. I am also used to the style of defining at the beginning of the class, but just trying to be inline with existing classes. May be we can start changing the style now, as easy to find all variables right at the beginning of the class.